### ISSUE TITLE
working_directory changing with each launch from Run button in IDE on macOS VM target

### ISSUE THREAD

**Issue by retupeltonen** [YoYoGames/GameMaker-Bugs#2275]
### Description

In previous versions, running the game from the Run button in the IDE targeting macOS VM, there was a persistent folder for saving and loading files from the game (working_directory). It was located in /Library/Application Support/com.yoyogames.macyoyorunner.

Now working_directory points to a different temp folder each time. The temp folders are generated in the folder where Settings - General Settings - Paths - Temp directory is pointing. The generated folders are named ProjectName_[randomizedstring]_VM and the string is different every time.

Beta IDE v2023.1100.0.444 Beta Runtime v2023.1100.0.455
[GameMakerBug.zip](https://github.com/YoYoGames/GameMaker-Bugs/files/13449817/GameMakerBug.zip)

### Expected Change

A persistent working_directory when running from the IDE on macOS VM

### Steps To Reproduce

1. Start GameMaker on macOS
2. Run a project from the Run button in the IDE on macOS VM target platform
3. Check to see working_directory
4. See the issue

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

Beta (Please put the exact version number in your Description)

### Which platform(s) are you seeing the problem on?

macOS

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by tabularelf** [comment]
I was just doing some double checking of my own, seems that this is mainly happening for GameMaker MacOS IDE -> MacOS VM. 
Windows IDE -> MacOS VM is unaffected. (`working_directory` gets sent to `/tmp/GameMakerStudio2-Beta/GMS2TEMP/project_name_#######_VM/`)

**Comment by rwkay** [comment]
OK I was looking at this and I don't think that there is a bug here... the `working_directory` is the current working directory of the process and that will be different on each run - it is set by starting the process. There has been no change of behaviour in this version from previous ones (that I can tell) NOTE: There was a change in earlier 2023.1100 runtimes that has now been fixed.

Now I think that what you really want to do is remove the `working_directory` from the start and either use the save directory which is called `game_save_id` see https://manual.yoyogames.com/GameMaker_Language/GML_Reference/General_Game_Control/game_save_id.htm or you want to have no prefix at all (as that will always be the save area it saves too).

In short `working_directory` will move around and can be changed by calling processes so could be dangerous to use all the time.
